sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -769,7 +769,11 @@ def peek_messages(
         if timeout is not None and timeout <= 0:
             raise ValueError("The timeout must be greater than 0.")
         if not sequence_number:
-            sequence_number = self._last_received_sequenced_number or 1
+            sequence_number = (
+                self._last_received_sequenced_number + 1
```

Comment:
just to make sure we are sure this is exclusive and not inclusive (self._last +1 vs self._last)

Reply:
its inclusive, which is why subsequent calls to `peek_messages` would bring back results starting from seq id 1. 

so if your queue has 4 messages with sequence numbers ( 1, 2, 3, 4)
first peek with max_message_count set to 2, would bring 1, 2
second call to peek with max_message_count set to 2 would set sequence id to 3 so result will be 3, 4

also this https://github.com/Azure/azure-sdk-for-net/blob/0eb66bbec3b2a3618aa66e5a66fb5cfc864e0745/sdk/servicebus/Microsoft.Azure.ServiceBus/src/Core/MessageReceiver.cs#L825

Reply:
also this https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-amqp-request-response#peek-message